card/config/initializers/02_patches/active_record.rb

```diff
@@ -104,5 +104,18 @@ def match string
         end
       end
     end
+
+    module Migration
+      module ClassMethods
+        def check_pending!(connection = ::ActiveRecord::Base.connection)
+          [:structure, :core_cards, :deck, :deck_cards].each do |migration_type|
```

Comment:
just to have written this somewhere, I think the better names would be `core_structure`, `core_cards`, `deck_structure`, and `deck_cards`. 

But to make this work, we might have to migrate the name of the schema migration tables :)